### PR TITLE
MH-13541 Remove inclusion of non-existent scripts in Admin UI

### PR DESCRIPTION
**Pull request by staubesv**
When starting the Admin UI with "grunt serve" or "grunt proxy", there are 404 NOT FOUND errors reported in the browser console.

Analysis:
The scripts referenced in index.html indeed to not exists.

### CODE REVIEW

File: modules/admin-ui/src/main/webapp/index.html [hunk 1]
```diff
@@ -318,7 +316,6 @@
     <script src="scripts/shared/controllers/navigationController.js"></script>
     <script src="scripts/shared/controllers/applicationController.js"></script>
     <script src="scripts/shared/controllers/applicationController.js"></script>
-    <script src="scripts/shared/controllers/userPreferencesController.js"></script>
```
Comment (KatrinIhler) [inline modules/admin-ui/src/main/webapp/index.html:321]: applicationController is a duplicate, mind kicking one of them out?

File: modules/admin-ui/src/main/webapp/index.html [hunk 2]
```diff
@@ -146,7 +146,6 @@
     <script src="scripts/shared/services/services.js"></script>
     <script src="scripts/shared/services/resourceHelperService.js"></script>
     <script src="scripts/shared/services/authService.js"></script>
-    <script src="scripts/shared/services/cryptService.js"></script>
```
Comment (KatrinIhler) [inline modules/admin-ui/src/main/webapp/index.html:149]: The cryptservice is also included in login-html, would you mind removing it there as well?

**Review by KatrinIhler — CHANGES_REQUESTED** [review]
Looks good overall, just minor stuff.

File: modules/admin-ui/src/main/webapp/index.html (hunk 2 above)
Comment (staubesv) [reply modules/admin-ui/src/main/webapp/index.html:149]: Done.

File: modules/admin-ui/src/main/webapp/index.html (hunk 1 above)
Comment (staubesv) [reply modules/admin-ui/src/main/webapp/index.html:321]: Done.

File: modules/admin-ui/src/main/webapp/login.html [hunk 3]
```diff
@@ -84,11 +84,6 @@
     <!-- endbower -->
     <!-- endbuild -->
 
-    <!-- build:js(src/main/webapp) scripts/vendor.js -->
```
Comment (staubesv) [inline modules/admin-ui/src/main/webapp/login.html:87]: @KatrinIhler Found even more ;-)

Comment (KatrinIhler) [reply modules/admin-ui/src/main/webapp/login.html:87]: Nice!

**Review by KatrinIhler — APPROVED** [review]

**Pull request merged**